Runtime/Integrations/Steam/SteamLobby.cs

```diff
@@ -96,9 +100,9 @@ internal class SteamLobby : Lobby {
 
   public override unsafe void SendLobbyMessage(ReadOnlySpan<byte> msg) {
     fixed (byte* ptr = msg) {
-        if (!_lobby.SendChatBytes(ptr, msg.Length)) {
-            Debug.LogError("Failed to send Steam Lobby Packet.");
-        }
+        //if (!_lobby.SendChatBytes(ptr, msg.Length)) {
+        //    Debug.LogError("Failed to send Steam Lobby Packet.");
+        //}
```

Comment:
Any reason this was commented out?

Reply:
Sending chat bytes are no longer supported through facepunch. I'm just going to throw a NotImplemented error then.